### PR TITLE
Issue 16 paypal donation component

### PR DESCRIPTION
**Pull request by MoeBuTa**
## Change Summary
**[Briefly summarise the changes that you made. Just high-level stuff]**

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [x] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

### CODE REVIEW

File: src/components/paypal/paypal.tsx
Comment (cjnbennett) [inline src/components/paypal/paypal.tsx:1]: Will have to integrate selecting the donation amount in the frontend at some point, but good MVP code to get us started!

File: tailwind.config.js
Comment (cjnbennett) [inline tailwind.config.js:1]: These styles are already defined in src/styles/globals.css - use these instead

**Review by cjnbennett — CHANGES_REQUESTED** [review]
Great work! There's still a lot of room for integrating this into the frontend more (though perhaps this should be a separate issue), but I'm keen to merge this code into main. I just have a couple of minor requests before merging:
- Clean up the comments - don't use comments unless they're necessary (good example of where to use them is in a function docstring)
- Use the styles defined in src/styles/globals.css rather than modifying tailwind.config.js
- Rather than using svg tags to display images directly as you are right now, save the images to the public/images folder and reference them from there. (This makes the code cleaner). It doesn't matter if they're saved as pngs or are low quality, for now we care mostly about having some placeholders which we can replace with higher resolution versions later

**Comment by MoeBuTa** [comment]
> Great work! There's still a lot of room for integrating this into the frontend more (though perhaps this should be a separate issue), but I'm keen to merge this code into main. I just have a couple of minor requests before merging:
> 
> * Clean up the comments - don't use comments unless they're necessary (good example of where to use them is in a function docstring)
> * Use the styles defined in src/styles/globals.css rather than modifying tailwind.config.js
> * Rather than using svg tags to display images directly as you are right now, save the images to the public/images folder and reference them from there. (This makes the code cleaner). It doesn't matter if they're saved as pngs or are low quality, for now we care mostly about having some placeholders which we can replace with higher resolution versions later

Hi, i've resolved these issues in my local repo. do you wanna have a look in person?

**Review by cjnbennett — APPROVED** [review]

**Pull request merged**